### PR TITLE
Show protected branch rule names again

### PR DESCRIPTION
**Pull request by HesterG**
`!important`s for one of the primary label selectors are removed by #23774, so the repository branch protection settings ui will not have the demanding css. This PR modifies `.ui.primary.label` to fix it.

Before:
飞书20230404-115410

After:
截屏2023-04-04 11 56 32

### CODE REVIEW

File: web_src/css/base.css [hunk 1]
```diff
@@ -2383,9 +2383,9 @@ a.ui.label:hover {
 
 .ui.basic.labels .primary.label,
 .ui.ui.ui.basic.primary.label {
-  background: transparent;
-  border-color: var(--color-primary);
-  color: var(--color-primary);
+  background: transparent !important;
+  border-color: var(--color-primary) !important;
+  color: var(--color-primary) !important;
```
Comment (wxiaoguang) [inline web_src/css/base.css]: Please leave a comment about how it works.

Comment (HesterG) [reply web_src/css/base.css]: Added in https://github.com/go-gitea/gitea/pull/23907/commits/a806ea3240496818bb97db6ceee72394449b8ec7

**Comment by silverwind** [comment]
We could remove the `!important` on `.ui.primary.label` which also fixes it, but I' not sure if anything else would break.

**Comment by HesterG** [comment]
> We could remove the `!important` on `.ui.primary.label` which also fixes it, but I' not sure if anything else would break. I think it's worth a try.

I do find one specific place that will be overwtitten by semantic.css, this is in account setting of repository settings. But they look similar. 
截屏2023-04-04 18 09 31

**Comment by silverwind** [comment]
> I do find one specific place that will be overwtitten by semantic.css if removed the !important in .ui.primary.label, this is in account setting of site administration settings. But they look similar.

That's an easy fix, alter the selector to match the original Fomantic one, e.g. `.ui.ui.ui`.

**Comment by HesterG** [comment]
> > I do find one specific place that will be overwtitten by semantic.css if removed the !important in .ui.primary.label, this is in account setting of site administration settings. But they look similar.
> 
> That's an easy fix, alter the selector in `base.css` to match the original Fomantic one, e.g. `.ui.primary.labels .label, .ui.ui.ui.primary.label`.

Added a commit https://github.com/go-gitea/gitea/pull/23907/commits/3d97a428ffe3ec5438b3ad309c0164b55f2f44f6 to apply the changes.

**Review by silverwind — APPROVED** [review]

**Review by lunny — APPROVED** [review]

**Review by delvh — APPROVED** [review]

**Pull request merged**